packages/strapi-provider-email-mailgun/lib/index.js

```diff
@@ -11,35 +11,11 @@ const mailgunFactory = require('mailgun-js');
 
 /* eslint-disable no-unused-vars */
 module.exports = {
-  provider: 'mailgun',
-  name: 'Mailgun',
-  auth: {
-    mailgun_default_from: {
-      label: 'Mailgun Default From',
-      type: 'text',
-    },
-    mailgun_default_replyto: {
-      label: 'Mailgun Default Reply-To',
-      type: 'text',
-    },
-    mailgun_api_key: {
-      label: 'Mailgun API Key',
-      type: 'text',
-    },
-    mailgun_api_host: {
-      label: 'Mailgun API Host',
-      type: 'text',
-    },
-    mailgun_domain: {
-      label: 'Mailgun Domain',
-      type: 'text',
-    },
-  },
   init: config => {
     const mailgun = mailgunFactory({
-      apiKey: config.mailgun_api_key,
-      host: config.mailgun_api_host,
-      domain: config.mailgun_domain,
+      apiKey: config.apiKey,
```

Comment:
One thing we could do is to send the full config object instead so the user can use the full extend of the provider options and in the readme we can document what provider is used to know more about the other params. What do you think ?